monix-eval/shared/src/test/scala/monix/eval/TaskAsyncSuite.scala

```diff
@@ -156,29 +156,29 @@ object TaskAsyncSuite extends BaseTestSuite {
     assertEquals(f.value, Some(Success(10000)))
   }
 
-  test("Task.async0(allowContinueOnCallingThread = true) should execute") { implicit s =>
-    val task = Task.async0[Int]({ (ec, cb) =>
+  test("Task.async0() should execute") { implicit s =>
```

Comment:
I think all of the tests that used to be with `allowContinueOnCallingThread = true` are duplicated in this file now so we could remove them / revert to the state from before my change

Reply:
👍 will take care of it.